templates/ecc/questionnaire_list.html

```diff
@@ -18,12 +18,7 @@
         <div class="card" id="control-{{ control.id }}">
           <color-bar id="{{ control.id }}"></color-bar>
           <div class="card-header flex-column align-items-start">
-            {% if control.depositing_organization %}
-              <div class="page-title">Organisme interrogé : {{ control.depositing_organization }}</div>
-              <div class="card-title">Procédure : {{ control.title }}</div>
-            {% else %}
-              <div class="page-title">{{ control.title }}</div>
-            {% endif %}
+            <control-title :control="{{ control.data }}" :key={{ control.id }}></control-title>
```

Comment:
t'as besoin de key ici? C'est pas dans le cadre d'un v-for donc je suis pas sure... Est-ce que Vue a donné un warning? (bon au pire ca fait pas de mal)

Reply:
Comme il y a une boucle for, j'ai pris l'habitude de mettre une key. Mais je ne sais pas si c'est vraiment pertinent. Je laisse pour l'instant.